### PR TITLE
Extending Protobuf RPC client with tracing headers

### PR DESCRIPTION
**Pull request by minond**
@film42 mind taking a look at this one? I can make this repo public and set it up on travis after this goes out.

### CODE REVIEW

File: lib/protobuf/rpc/extensions/client.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module Protobuf
+  module Opentracing
+    module Extensions
+      module Client
+        extend Forwardable
+
+        def_delegator :@connector, :send_request, :send_traced_request
```
Comment (film42) [inline lib/protobuf/rpc/extensions/client.rb]: :-1: Let's not pollute the class. Let's manually call send request down below.

File: lib/protobuf/rpc/extensions/client.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+        def send_request
+          operation = "#{options[:service]}##{options[:method]}"
+          ::OpenTracing.start_active_span(operation) do
+            send_traced_request
```
Comment (film42) [inline lib/protobuf/rpc/extensions/client.rb]: Just call `super`?

Comment (minond) [reply lib/protobuf/rpc/extensions/client.rb]: Duh, `super` works just find here.

**Review by film42 — APPROVED** [review]
Before we release a new version to rubygems, let's get a follow-up PR that tests this behavior. Even if we stub out the `OpenTracing` class, it would be nice to see that `OpenTracing.start_active_span` is called, headers are sent over the wire, etc.

**Pull request merged**